### DISCUSSION TITLE
Design Aesthetic for Businesses

### DISCUSSION THREAD

**Discussion by apollodelphi702** [security-mountain/security-mountain.github.io#8] [Ideas]
For businesses the 'video game' aesthetic for the project may be off-putting as a 'lack of professionalism' or they may even misunderstand our target audience (which is businesses and personal alike).

Additionally, the personal design may be hard to make mobile responsive in terms of design, and some users or audiences may prefer a simpler design. So I propose a 'boring mode' design - whereby users can turn off the video game aesthetic (or it is turned off automatically) and replaced by a simpler but easy to read design.

**Comment by ElliottS200** [comment]
This could definitely be a great idea, if not in current version, in later version, I like the idea of a 'boring mode' as it will open up the site to a wider variety of users

**Comment by bceda** [comment]
I agree that businesses probably expect a more professional aesthetic, whereas users prefer a more interactive (video-game like) experience. I like the idea of a boring mode though for users, to give them choice.
